runtime/development/src/migrations.rs

```diff
@@ -10,7 +10,42 @@
 // MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 // GNU General Public License for more details.
 
+use sp_core::parameter_types;
+parameter_types! {
+	pub const CollatorReward: cfg_primitives::Balance = cfg_primitives::constants::CFG;
+	pub const AnnualTreasuryInflationPercent: u32 = 3;
+}
+
 /// The migration set for Development & Demo.
 /// It includes all the migrations that have to be applied on that chain.
-pub type UpgradeDevelopment1046 =
-	pallet_collator_selection::migration::v1::MigrateToV1<crate::Runtime>;
+pub type UpgradeDevelopment1046 = (
```

Comment:
Wow! a lot of migrations. How can we ensure this is done in the demo but not in dev? Is it because all of them are idempotent?